overview/makerdao.md

```diff
@@ -0,0 +1,24 @@
+{% hint style="warning" %} This documentation describes certain core aspects of MakerDAO's protocol that could change over time. Be aware that parts may be inaccurate or out of date. {% endhint %}
```

Comment:
General comment: This doc might need a bit of AI assistance to read more clearly.